must_triage/inspectors/ocp.py

```diff
@@ -78,3 +79,25 @@ def pod_ready(obj):
                 f"Container '{cs['name']}' in "
                 f"pod '{obj['metadata']['name']}' is not ready")
         return result
+
+    @staticmethod
+    def restart_count(obj):
+        result = list()
+        status = obj['status']
+        try:
+            container_statuses = status['containerStatuses']
+        except KeyError:
```

Comment:
If we hit this condition, the `pod_ready` check will have already done the bit below. Perhaps it would make more sense to see if 'containerStatuses` is present in `status`, and skip the check if it is not.